.travis.yml

```diff
@@ -7,11 +7,13 @@ python:
 # Specifying `branches.only = ['master']` can cause tagged builds to
 # not deploy. See travis-ci/travis-ci#2498 and travis-ci/travis-ci#1675.
 # We can fix this by only build master and branches/tags that follow the
-# format x.y.z.
+# format x.y.z. and also feature/issue-<num>, the latter is the style
+# used by HubFlow
 branches:
   only:
     - master
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
+    - /^feature\/issue-\d+$/
```

Comment:
I mentioned in contributing that we are using HubFlow, I think this is important.  As such, trusted developers working on feature branches in our repo should use this naming convention and, therefore, the branch will be built with Travis.  We may want to have a few other options here e.g. topic/issue-\d for example.